### DISCUSSION TITLE
Formatting on multi-line attribute value seems weird

### DISCUSSION THREAD

**Discussion by idlist** [vuejs/language-tools#2512] [Q&A]
I'm using TailwindCSS and it's common to have a very long class names. However, each time I format the multi-line class value, the indentation of the second line and below lines keeps increasing, like this:

![7NSY~%PSF_)`V%4T59X{8}9](https://user-images.githubusercontent.com/29368226/225330297-ebabeb41-f560-4375-b1fd-c5c1c6464eb8.png)

->

![D9QE9Y7` ~{M0J 7%C7H6`M](https://user-images.githubusercontent.com/29368226/225330550-a13e48a2-509f-47e7-82df-492d9b7aed9e.png)

->

![$~1 OT$DW16FL5)03WCG`DT](https://user-images.githubusercontent.com/29368226/225330611-62170e7b-e49d-4d11-b63c-7654013cbd19.png)

This problem does not appear in a pure HTML file, so I guess it's not some problem with VSCode itself. Also, setting `volar.format.initialIndent` to `false` would not cause the problem. Is it possibly to be some kind of bug, or has there been some solution/walkaround to this issue?

**Comment by johnsoncodehk** [comment]
Please track #2519.
